src/control/server/mgmt_system_test.go

```diff
@@ -914,7 +904,10 @@ func TestServer_MgmtSvc_StartRanks(t *testing.T) {
 
 					// set instance runner started and ready
 					ch := make(chan error, 1)
-					s.runner.Start(context.TODO(), ch)
+					if err := s.runner.Start(context.TODO(), ch); err != nil {
+						t.Logf("failed to start runner: %s", err)
+						return
```

Comment:
t.Fatalf?

Reply:
This is also called in a new goroutine, so we can't use `t.Fatalf()`. My thought was that if the runner fails to start, at least we've got a log message to understand why the test hung. Another option might be to `panic()`, but not sure if that would yield useful failure messages for debugging after the fact.